content/applications/inventory_and_mrp/inventory/management/misc/owned_stock.rst

```diff
@@ -1,45 +1,106 @@
-===============================
-Manage Stock that you don't own
-===============================
+=================================================
+Consignment: Buy and Sell Stock Without Owning It
+=================================================
 
-Sometimes, suppliers can offer you to store and sell products without
-having to buy those items. This technique is called *consignee stock*.
+Most of the time, products stored in a company's warehouse are either purchased from suppliers, or
+are manufactured in-house. However, suppliers will sometimes let companies store and sell products
+in their warehouses without having to buy those items up front. This is called **Consignment**.
 
-Consignee stock is a great way for manufacturers and suppliers to launch
-new products. As resellers may be reluctant to buy a product they are
-not sure to be able to sell, consignee stock will allow them to offer an
-item to check its market without having to pay for it in the first
-place.
-
-Of course, Odoo has the ability to manage those consignee stocks through
-advanced settings.
+This is a great way for suppliers to launch new products and easily get them out to their
+customers, and a great way for the company storing the products (the consignee) to earn sommething
+back for it. Consignees can even charge a fee for the convenience of storing products that they
+don't actually own.
 
 Configuration
 =============
 
-To use this feature, go to :menuselection:`Inventory --> Configuration --> Settings` in
-the inventory app. Then, enable the *Consignment* feature in the
-*Traceability* section. Now, hit save.
+To receive, store, and sell consignment stock, the *Consignment* feature needs to be enabled. To
+enable this, go to :menuselection:`Inventory --> Configuration --> Settings`, and under the
+**Traceability** section, check the box next to :guilabel:`Consignment`.
+
+.. image:: owned_stock/owned-stock-enable-consignment.png
+   :align: center
+   :alt: Enabled Consignment setting in Inventory configuration.
+
+Receive and Store Consignment Stock
+===================================
+
+With the *Consignment* feature enabled, consignment stock can now be received into a warehouse.
```

Comment:
With the :guilabel:`Consignment` feature enabled, consignment stock can now be received into a warehouse.